ocaml/xapi/xapi_fist.ml

```diff
@@ -23,16 +23,30 @@ open D
 
 (** {2 (Fill in title!)} *)
 
+let prefix = "/tmp/fist_"
+
 let fistpoint name =
   try
-    Unix.access ("/tmp/fist_" ^ name) [Unix.F_OK] ;
+    Unix.access (Printf.sprintf "%s%s" prefix name) [Unix.F_OK] ;
```

Comment:
The same code is used below. This suggest a function `path_for: string -> string` would be useful that constructs the filename and that has `prefix` as a local binding. Now you would have `Unix.access (path_for name) [Unix.F_OK]`